samples/Meeting/CHANGELOG.json

```diff
@@ -0,0 +1,190 @@
+{
+  "name": "calling",
```

Comment:
This is probably one of those instances of calling we want to remove? Do we need a changelog for this specific sample so far when this is the only change?

Reply:
Feel free to disregard but this changelog looks like it was copied and pasted from the calling sample

Reply:
yeah I think the changelog got snuck in there I will remove them since this should have its own in the future